File: src/test/java/com/binance/api/examples/MarginGeneralEndpointsExample.java
package com.binance.api.examples;

import com.binance.api.client.BinanceApiClientFactory;
import com.binance.api.client.BinanceApiMarginRestClient;
import com.binance.api.client.domain.general.MarginPair;

import java.util.List;

/**
 * Examples on how to use the general endpoints.
 */
public class MarginGeneralEndpointsExample {

    public static void main(String[] args) {
        BinanceApiClientFactory factory = BinanceApiClientFactory.newInstance("YOUR_API_KEY", "YOUR_SECRET");
        BinanceApiMarginRestClient client = factory.newMarginRestClient();

        // Exchange info
        List<MarginPair> marginPairs = client.getAllPairs();
        System.out.println(marginPairs);
    }
}

File: src/test/kotlin/com/binance/api/examples/MarginGeneralEndpointsExample.kt
package com.binance.api.examples

import com.binance.api.client.BinanceApiClientFactory.Companion.newInstance
import com.binance.api.client.domain.general.MarginPair

/**
 * Examples on how to use the general endpoints.
 */
class MarginGeneralEndpointsExample {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val factory = newInstance("YOUR_API_KEY", "YOUR_SECRET")
            val client = factory.newMarginRestClient()

            // Exchange info
            val marginPairs: List<MarginPair> = client.getAllPairs()
            println(marginPairs)
        }
    }
}